README.md

```diff
@@ -1 +1,16 @@
-# matrix_check_sum
\ No newline at end of file
+# Matrix: Check sum
+Checks if the sum of values in a given row matches the sum of values in the corresponding column.
+
+## Exercise
+* Design and implement a method that checks that for the given matrix, where number of rows are equal to number of columns whether the sum of each row matches the sum of the corresponding column i.e. sum
+of numbers in row 'i' is the same as the sum of numbers in column 'i' for 'i' ranging from '0' to 'row.length-1'. If this is the case, the method return true. Otherwise, it return false.
+* For example, consider the input as:
```

Comment:
We need to use code blocks for this matrix to render properly in Markdown:
```
1 2 3 4
9 5 3 1
0 3 5 6
0 8 3 6
```

Reply:
Done.